pipelines/pr.yaml

```diff
@@ -23,13 +25,37 @@ jobs:
       # the specific flavors below that are experiencing failures in CI (for example,
       # if we're actively working on features that have a lot of churn on underlying
       # WMR APIs, buildUWPArm would be a good candidate to re-enable)
+      buildStandalone: true
       buildUWPX86: false
       buildUWPArm: false
       buildUWPDotNet: false
       # For mrtk_pr builds, validation scripts (code and docs) should be scoped to only
       # the set of changed files, so that we can save some time. There's no need to
       # check unchanged files for code style/doc style violations.
       scopedValidation: true
+      # This particular job shouldn't run tests (they happen in PRValidationTest)
+      runTests: false
       UnityVersion: $(Unity2018Version)
 
   - template: templates/end.yml
+- job: PRValidationTest
+  timeoutInMinutes: 20
+  pool:
+    name: On-Prem Unity
+    demands:
+    - Unity2018.4.6f1
+    - COG-UnityCache-WUS2-01
+    - SDK_18362 -equals TRUE
+  steps:
+  - template: templates/common.yml
+    parameters:
+      # The only task that this job should do is run the tests.
+      # Building is handled by PRValidationBuild.
+      runTests: true
+      buildStandalone: false
+      buildUWPX86: false
+      buildUWPArm: false
+      buildUWPDotNet: false
+      validateCode: false
```

Comment:
I wonder if we want to have scoped validation run on this job too. It's a pretty quick run (on the order of seconds) that could serve as a quick out for both jobs in case there's an initial issue. Otherwise, we're running tests on a PR that's going to be updated and have tests run on it again before merging.

Reply:
Of course, on the other hand it'd show up as a failure for both jobs, when it's really the same single failure.